src/main/java/uk/gov/register/util/RegisterComparer.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.register.util;
+
+import uk.gov.register.core.RegisterMetadata;
+
+import java.util.List;
+import java.util.Objects;
+
+public class RegisterComparer {
+
+    public static boolean equals(RegisterMetadata registerMetadata1, RegisterMetadata registerMetadata2) {
+
+        return registerMetadata1.getRegisterName().equals(registerMetadata2.getRegisterName()) &&
```

Comment:
Potentially drop the `&&` down to the beginning of line 13 so that everything reads well